### PR TITLE
Update dockerhub description [semver:minor]

### PR DESCRIPTION
**Pull request by iynere**
https://github.com/CircleCI-Public/docker-orb/pull/47

adding tests before merging

### CODE REVIEW

**Comment by ArmaanT** [comment]
Bump on this. Is there anything I can do to help get this merged?

File: src/jobs/publish.yml [hunk 1]
```diff
@@ -98,6 +103,11 @@ parameters:
     type: boolean
     default: true
 
+  update-description:
+    description: Update the image description on Docker Hub?
+    type: boolean
+    default: true
```
Comment (eddiewebb) [inline src/jobs/publish.yml]: default false to be backward compatible and inline with default behavior of docker hub push

File: src/examples/build-without-updating.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+description: >
```
Comment (eddiewebb) [inline src/examples/build-without-updating.yml]: flip this example to be new example of adding the additional feature instead of disabling it.

**Comment by gmemstr** [comment]
Pipeline failed due to rate limiting :( But the test for updating the description appears to have worked.

**Comment by gmemstr** [comment]
I agree, actually - that's a much clearer name for it!

**Pull request merged**